Gemfile.jruby-3.1.lock.release

```diff
@@ -322,7 +322,7 @@ GEM
       lru_redux (~> 1.1.0)
     logstash-filter-drop (3.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-elastic_integration (8.16.0-java)
+    logstash-filter-elastic_integration (8.16.1-java)
       logstash-core (>= 8.7.0)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
     logstash-filter-elasticsearch (3.16.2)
```

Comment:
Can we please update this to `3.17.0` to pick up the custom headers addition to the Elasticsearch filter
```suggestion
    logstash-filter-elasticsearch (3.17.0)
```

and also `logstash-input-elasticsearch` to `4.21.1` for the same